dynamo/plot/utils.py

```diff
@@ -518,9 +533,10 @@ def _matplotlib_points(
         else:
             main_debug("drawing without frontiers and contour")
             main_debug("using cmap: %s" % (str(cmap)))
-            ax.scatter(
-                points[:, 0],
-                points[:, 1],
+            _scatter_projection(
+                ax,
+                points,
+                projection,
```

Comment:
should we also add rasterized=rasterized, here?

Reply:
The old code does not have rasterized=rasterized here. I am not sure if I add some argument to ~2000 lines of code function "scatters" can cause any problem.